molgenis-data-rest/src/main/java/org/molgenis/data/rest/exception/IncompatibleValueTypeException.java

```diff
@@ -48,6 +48,6 @@ public String getMessage()
 	@Override
 	protected Object[] getLocalizedMessageArguments()
 	{
-		return new Object[] { attribute.getName(), type, String.join(",", expectedTypes) };
+		return new Object[] { attribute.getName(), type, String.join("or", expectedTypes) };
```

Comment:
`" or "` with the spaces me thinks

Reply:
Also, `or` isn't localized here. If we really want to do that I guess we'd have to add another message formatter?

Reply:
or a key for the delimiter